t4c/Dockerfile

```diff
@@ -34,6 +34,7 @@ COPY export.py /opt/scripts/export.py
 # Install T4C
 COPY ./updateSite/$CAPELLA_VERSION /opt/updateSite
 WORKDIR /opt/updateSite
+# hadolint ignore=SC2046
```

Comment:
Regarding all three `# hadolint ignore=SC2046` statements. The rule makes sense and the commands would fail if the files contain whitespaces. Can you rework all three occurrences of `SC2046`?

In this case, the command can be replaced by: `find /opt/updateSite -type f -name "*.zip" -exec chmod +r {} \;`

Reply:
I left out refactorings on purpose. It should be little work to refactor those, though.